src/test/java/org/broadinstitute/hellbender/tools/walkers/RevertQualityScoresIntegrationTest.java

```diff
@@ -0,0 +1,59 @@
+package org.broadinstitute.hellbender.tools.walkers;
+
+import htsjdk.samtools.SAMRecord;
+import htsjdk.samtools.SamReader;
+import htsjdk.samtools.SamReaderFactory;
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.broadinstitute.hellbender.exceptions.GATKException;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.io.IOException;
+
+public class RevertQualityScoresIntegrationTest extends CommandLineProgramTest {
+    @Test
+    public void testRevertQualityScores() throws IOException {
+        final File inputBam = new File(getTestDataDir(), "BQSR/expected.HiSeq.1mb.1RG.2k_lines.alternate.recalibrated.DIQ.bam");
```

Comment:
Don't use an expected output bam directly as input for this test (as expected outputs can change over time). Better to make a copy of this bam that's exclusive to this test.